en/docs/edit-the-code/code-actions.md

```diff
@@ -1 +1,70 @@
-# Code actions
\ No newline at end of file
+# Code actions
+
+## Types of Code actions
```

Comment:
We don't need this 2nd level heading. Instead let's just say
```
In VS Code, Code Actions can provide both refactorings and Quick Fixes for detected issues
```
(copied from vscode docs)